aries-site/src/data/wcag/components.json

```diff
@@ -861,6 +861,518 @@
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.4.10",
+        "label": "Section Headings",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.4.11",
+        "label": "Focus Not Obscured (Minimum)",
+        "status": "failed",
+        "notes": "TextInput with suggestions, the treatment of the suggestion that is in focus is not meeting requirement."
+      },
+      {
+        "rule": "2.4.12",
+        "label": "Focus Not Obscured (Enhanced)",
+        "status": "failed",
+        "notes": "TextInput with suggestions, the treatment of the suggestion that is in focus is not meeting requirement."
+      },
+      {
+        "rule": "2.4.13",
+        "label": "Focus Appearance",
+        "status": "failed",
+        "notes": "TextInput with suggestions, the treatment of the suggestion that is in focus is not meeting requirement."
+      },
+      {
+        "rule": "2.5.1",
+        "label": "Pointer Gestures",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.5.2",
+        "label": "Pointer Cancellation",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "2.5.3",
+        "label": "Label in Name",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "2.5.4",
+        "label": "Motion Actuation",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.5.5",
+        "label": "Target Size (Enhanced)",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "2.5.6",
+        "label": "Concurrent Input Mechanisms",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "2.5.7",
+        "label": "Dragging Movements",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.5.8",
+        "label": "Target Size (Minimum)",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "3.1.1",
+        "label": "Language of Page",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "3.1.2",
+        "label": "Language of Parts",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "3.1.3",
+        "label": "Unusual Words",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "3.1.4",
+        "label": "Abbreviations",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "3.1.5",
+        "label": "Reading Level",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "3.1.6",
+        "label": "Pronunciation",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "3.2.1",
+        "label": "On Focus",
+        "status": "passed",
```

Comment:
I think this might be failing because in the default suggestion example when the TextInput receives focus, focus is then moved to the default suggestion which creates a change of context

Reply:
When I inspect the active element in the DOM, it's still on the input, even though the suggestion gets "active" styling.

<img width="1329" alt="Screenshot 2025-03-18 at 12 17 09 PM" src="https://github.com/user-attachments/assets/fa66084c-2313-44a7-9234-f79f1f849476" />

Reply:
Interesting, I'm surprised the screenreader is announcing 'suggestion 2' if the focus doesn't change. It still feels like something isn't quite right here. When a screenreader user moves to the textinput they don't get information about the input field and instead the screenreader jumps to reading out 'suggestion 2'. I'd imagine a screenreader user would have a hard time figuring out what's happening here

Reply:
Okay that's fair. I'll mark as failing also.